app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -151,25 +198,17 @@ public void showScanButton(boolean show) {
 
     @Override
     public void showAttendees(List<Attendee> attendees) {
-        // The list is loaded from presenter, so we just need
-        // to notify RecyclerView to update the data
-        attendeeListAdapter.notifyDataSetChanged();
+        itemAdapter.clear();
+        itemAdapter.add(attendees);
     }
 
     @Override
     public void updateAttendee(int position, Attendee attendee) {
-        // The attendee is saved correctly in list by presenter, so we
-        // just need to notify RecyclerView that an item has changed
-        if(position == -1) {
-            attendeeListAdapter.notifyDataSetChanged();
-            return;
-        }
-        attendeeListAdapter.notifyItemChanged(position);
+        itemAdapter.set(position, attendee);
```

Comment:
Position can be -1 if it can't find attendee. Please add checks as previously done